VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+* **Non-voting:** If a validator did not vote on a proposal and once the fault is reported by a someone, his stake will be a little slashed. He will also get a penalty. Once a validator gets a penalty, no additional penalty can be submitted for a given period (3 weeks). After this period, if a second penalty is submitted, the validator gets a second light slash and gets unbonded. If, after the first penalty, no other penalty is submitted for a 3 months period, then the penalty can be cleared by sending a special transaction.
+
+Note that even if a validator does not intentionally misbehaves, it can still be slashed if its node crashes, looses connectivity, gets DDOSed or if its private keys are compromised.
+
+### Incentives for validator #101 to stake?
+
+Validator candidates that do not have enough total stake to be in the validator set will have to run validator operation (or at least be ready to) without getting any revenue. 
+
+There are solutions that are considered to mitigate this issue. For example, we could create a sub-layer of delegation. Validator candidate could delegate to existing validators while they’re trying to attract delegators, thereby generating revenue. At the same time, delegators already bonded to validators could signal their intention to switch to this validator candidate should it get enough stake to enter the validator set. When the validator candidate would get enough pledge, it would transition to a validator and all the delegator who pledged would join its staking pool.
+
+### Do validators need to self-bond ATOMs?
+
+Delegators should want their validator to have self-bonded ATOMs in their staking pool; that is, bonded ATOMs of their own. In other words, validators should have skin-in-the-game. 
+
+In order for delegators to have some guarantee about how much skin-in-the-game their validator has, the latter must signal a minimum amount of self-bonded ATOMs. If a validator's self-bond goes below the limit that it predefined, this delegator and all of its delegators will unbond. 
+
+### How to prevent concentration of stake in the hands of a few top validators?
+
+For now we are expecting the community to behave in a smart and self-preserving way. When a mining pool in Bitcoin gets too much mining power the community usually stops contributing to that pool. We want to harness the same effect with Cosmos initially. We will put several mechanisms in place to smoothen this process as much as possible:
+
+* **Fast switching:** We want to allow delegators to easily switch from one validator to the other, in order to reduce validator stickiness 
+* **Hack bounty:** We want to incentivize the community to hack validators. There will be bounties proportionate to the size of the validator, so that a validator becomes bigger of a target as its stake grows.
+* **UI warning:** Users will be warned by the wallet’s UI if they want to delegate to a validator that already has a significant amount of staking power
```

Comment:
This entire section should not reference "we" so much or at all, please rephrase when possible

Reply:
addressed